ceph.spec.in

```diff
@@ -121,6 +121,7 @@ BuildRequires:	pkgconfig
 BuildRequires:	python
 BuildRequires:	python-devel
 BuildRequires:	python-nose
+BuildRequires:	python-prettytable
```

Comment:
so the consumer of this line is install-deps.sh actually. i am far from an expert of `dnf` or its builddep plugin. so guess this is the best way to document the "make check" dependencies at this moment.